BentoKit/BentoKit/Views/Accessory.swift

```diff
@@ -42,6 +42,10 @@ public final class AccessoryView: InteractiveView {
     }
 
     public override var intrinsicContentSize: CGSize {
+        if case let .custom(view) = accessory {
+            view.layoutIfNeeded()
```

Comment:
I think if `view` would pinned to the edges not to `centerInSuperview` them you would not need that

Reply:
what about width?
Actually I think there needs to be an adjustment for the height calculation as it currently has some constant `24`. Though it seems to be working in playground correctly.

Reply:
Why isn't the intrinsic content size of the custom view be used? If one needs an accessory that cannot have a definitive intrinsic content size, it sounds something is off.

Reply:
Fair enough.